### ISSUE TITLE
Specify what we mean by private attributes

### ISSUE THREAD

**Issue by JJ** [perl6/doc#1965]
## The problem

As seen in [this SO question](https://stackoverflow.com/questions/50031400/inheriting-private-attributes-in-perl-6)

## Suggestions

Expand the sentence in [the Attributes section](https://docs.perl6.org/language/objects#Attributes) explaining what we mean by private.

**Comment by fluca1978** [comment]
I don't agree. Any OOP language I know of defines the concept of "private" attributes as "visible only within the class", so I don't believe we should elaborate any more.
Maybe pointing to an external resource, like <https://en.wikipedia.org/wiki/Object-oriented_programming#Encapsulation>.

**Comment by JJ** [comment]
Well, that would be elaborating :-)

My point is that if there's a question in Stackoverflow, some kind of
clarification (including pointing to an external resource) might be needed.

**Comment by fluca1978** [comment]
Agree on that, but instead of clarifying what `private` means we should place an introduction paragraph on the very beginning of the page (?) saying that Perl 6 _does_OOP and encapsulation, so refer to the external resource. Otherwise we risk any question will make us add some statement over here and there.

I can do it, in the case assign the ticket.

**Comment by JJ** [comment]
Whatever you do is is fine for me :-) Self-assign it, or if you don't have
the time or whatever, assign it to me.
